stable/elastalert/templates/config.yaml

```diff
@@ -14,11 +14,21 @@ data:
     scan_subdirectories: false
     run_every:
       minutes: {{ .Values.runIntervalMins }}
+{{- if .Values.elasticsearch.realertIntervalMins }}
+    realert:
+      minutes: {{ .Values.elasticsearch.realertIntervalMins }}
+{{- end }}
     buffer_time:
       minutes: {{ .Values.bufferTimeMins }}
     es_host: {{ .Values.elasticsearch.host }}
     es_port: {{ .Values.elasticsearch.port }}
+{{- if .Values.elasticsearch.username }}
+    es_username: {{ .Values.elasticsearch.username }}
+{{- end }}
+{{- if .Values.elasticsearch.password }}
+    es_password: {{ .Values.elasticsearch.password }}
+{{- end }}
     writeback_index: {{ .Values.writebackIndex }}
     use_ssl: {{ .Values.elasticsearch.useSsl }}
     alert_time_limit:
-      days: 2
+      minutes: {{ Values.alertRetryLimitMins }}
```

Comment:
Should be .Values with leading period/dot.

Reply:
Oh man! Thanks not sure how I missed that, must have removed after I installed this. Thanks!